src/backend/InvenTree/exporter/models.py

```diff
@@ -0,0 +1,45 @@
+"""Data export model definitions."""
+
+from django.contrib.auth.models import User
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class ExportOutput(models.Model):
```

Comment:
Question: This is another file-centered model; would it make sense to refactor how we handle files to unify them or should that be handled separately?